src/practices/Python/PythonLinterUsedPractice.ts

```diff
@@ -0,0 +1,29 @@
+import { IPractice } from '../IPractice';
+import { PracticeEvaluationResult, PracticeImpact, ProgrammingLanguage } from '../../model';
+import { DxPractice } from '../DxPracticeDecorator';
+import { PracticeContext } from '../../contexts/practice/PracticeContext';
+
+@DxPractice({
+  id: 'Python.LinterUsedPractice',
+  name: 'Use a Python Linter Dependency',
+  impact: PracticeImpact.medium,
+  suggestion: 'Use a linter for clean and error-free development experience.',
+  reportOnlyOnce: true,
+  url: 'https://developerexperience.io/practices/linting',
+})
+export class PythonLinterUsedPractice implements IPractice {
+  async isApplicable(ctx: PracticeContext): Promise<boolean> {
+    return ctx.projectComponent.language === ProgrammingLanguage.Python;
+  }
+
+  async evaluate(ctx: PracticeContext): Promise<PracticeEvaluationResult> {
+    if (!ctx.packageInspector) {
+      return PracticeEvaluationResult.unknown;
+    }
+
+    if (ctx.packageInspector.hasOneOfPackages(['pylint', 'pyflakes', 'flake8-bugbear', 'flake8', 'pydiatra', 'it'])) {
```

Comment:
I recommend adding a `pycodestyle`. The `flake8-bugbear` is just a plugin, so I think the `flake8` is sufficient. I don't think `pydiatra` should be checked. The Readme is pretty low quality so I wouldn't recommend using that. I couldn't find the `it`. What is it? :)

Reply:
I just checked out pycodestyle thoroughly I thought it was just a stylistic linter, therefore, didn't include it but it seems it also does some logical check so seems like a good linter. 
it -> [github repo](https://github.com/thg-consulting/it)
pydiatra seems to have a lot of features but yeah the documentation isn't so yeah better to remove it.
flake8-bugbear is a plugin but it can work on its own like pycodestyle or pyflakes too.

Reply:
It - I think we should check just some bigger and more used linters. I would remove it.
flake8-bugbear - if it's a plugin I don't think it's a good practice to use it as a stand-alone linter. I would remove it too.
I hope I didn't disappoint you with my requests. ❤️

Reply:
Not all seems logical to not use these also I was thinking can we also include autopep8 as its quite widely used as pycodestyle and they are just the same thing. what do you think :) ?

Reply:
Oh, sorry, I missed your comment. I checkech the autopep8 and it's just a code formatted which uses pycodestyle. It's quite the same relationship between eslint end prettier in JS. So this one shouldn't be checked in practice for linters :) . But it can be used for another practice which checks some code formatting package :)

Reply:
okay cool made the changes